### ISSUE TITLE
int_all_dag_relationships fails for Redshift target without schema bound view

### ISSUE THREAD

**Issue by JCWahoo** [dbt-labs/dbt-project-evaluator#179]
### Describe the bug
int_all_dag_relationships fails when loading a Redshift target configured with +bind: false in dbt_project.yml

### Steps to reproduce
Execute a build with a Redshift target and schema bound views disabled

### Expected results
Target object is created

### Actual results
int_all_dag_relationships fails

### Screenshots and log output

Database Error in model int_all_dag_relationships (models/marts/core/int_all_dag_relationships.sql)
  All the relation names inside should be qualified when creating VIEW WITH NO SCHEMA BINDING.
  compiled SQL at target/run/dbt_project_evaluator/models/marts/core/int_all_dag_relationships.sql

### System information
packages:
  - package: dbt-labs/dbt_utils
    version: 0.8.6
      
  - package: dbt-labs/redshift
    version: 0.6.1

  - package: dbt-labs/codegen
    version: 0.7.0

  - package: data-mie/dbt_profiler
    version: 0.4.1

  - package: elementary-data/elementary
    version: 0.4.6

  - package: dbt-labs/dbt_project_evaluator
    version: 0.1.1

**Which database are you using dbt with?**

- [X ] redshift

**The output of `dbt --version`:**
```
dbt Cloud v1.1.57.28
```

### Are you interested in contributing the fix?
Possibly, would definitely need a hand

**Comment by b-per** [comment]
Thankds for the report @JCWahoo . We'll have a look and add an integration test about it as well!

**Comment by b-per** [comment]
I was able to reproduce it on my side but it looks like a Redshift bug to me. The only relation we read from is `int_direct_relationships` and it is actually already qualified. I'm wondering if this is not a problem with the support of recursive CTEs in non-binding views.

Anyway, `master` already contains a fix for that as we are moving `int_all_dag_relationships` to a table.

You can wait for version 0.1.2 that should be out in the next days, or in the meantime, add the following to your `dbt_project.yml`:
```
models:
  dbt_project_evaluator:
    marts:
      core:
        int_all_dag_relationships:
          +materialized: table
```

**Comment by JCWahoo** [comment]
Thanks! Appreciate the quick turnaround looking forward to checking it out

**Comment by b-per** [comment]
This should be fixed in the new `1.2` release
